src/test/ui/specialization/specialization-default-types.stderr

```diff
@@ -21,7 +21,7 @@ LL |     Example::generate(t)
    |
    = note:       expected struct `std::boxed::Box<T>`
            found associated type `<T as Example>::Output`
-   = note: consider constraining the associated type `<T as Example>::Output` to `std::boxed::Box<T>`
+   = help: consider constraining the associated type `<T as Example>::Output` to `std::boxed::Box<T>`
```

Comment:
We should eventually handle the case of `#![feature(specialization)]`, but I won't on this PR.